File: js/viaje.js
class Viaje {
    constructor(id = null, usuario = null, ticket = null, turno = null, ubicacion = null, fecha = null, conductor = null) {
        this._id = id;
        this._usuario = usuario;
        this._ticket = ticket;
        this._turno = turno;
        this._ubicacion = ubicacion;
        this._fecha = fecha;
        this._conductor = conductor;
    }
    get id() {
        return this._id;
    }

    set id(id) {
        this._id = id;
    }

    get usuario() {
        return this._usuario;
    }

    set usuario(usuario) {
        this._usuario = usuario;
    }

    get ticket() {
        return this._ticket;
    }

    set ticket(ticket) {
        this._ticket = ticket;
    }

    get turno() {
        return this._turno;
    }

    set turno(turno) {
        this._turno = turno;
    }

    get ubicacion() {
        return this._ubicacion;
    }

    set ubicacion(ubicacion) {
        this._ubicacion = ubicacion;
    }

    get fecha() {
        return this._fecha;
    }

    set fecha(fecha) {
        this._fecha = fecha;
    }

    get conductor() {
        return this._conductor;
    }

    set conductor(conductor) {
        this._conductor = conductor;
    }

}

File: App_Code/Viaje.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Runtime.Serialization;
using System.Web;
using System.Xml.Serialization;

namespace es.itsduero.iecscyl
{
    [DataContract]
    public class Viaje
    {
        [XmlElement("ViajeUsuario")]
        public Usuario usuario { get; set; }
        [XmlElement("TViajeTicket")]
        public Ticket ticket { get; set; }
        [XmlElement("ViajeID")]
        public int ID { get; set; }
        [XmlElement("ViajeTurno")]
        public short turno { get; set; } //0 descanso, 1 mañanas, 2 tardes, 3 noches
        [XmlElement("ViajeUbicacion")]
        public Ubicacion ubicacion { get; set; }
        [XmlElement("ViajeFecha")]
        public DateTime fecha { get; set; }
        [XmlElement("ViajeConductor")]
        public Usuario conductor { get; set; }
    }
}